function-composition/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Function Composer
+category: Behavioral
+language: en
+tag:
+  - Functional Programming
+  - Composition
+  - Java
+---
+
+## Name / classification
```

Comment:
this section not needed, see https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template